tests/ClientTest.php

```diff
@@ -107,6 +107,42 @@ public function testCaptureException(): void
         $client->captureException($exception);
     }
 
+    /**
+     * @dataProvider captureExceptionWithExcludedExceptionsDataProvider
+     */
+    public function testCaptureExceptionWithExcludedExceptions(bool $shouldCapture, string $excluded, \Throwable $thrown): void
+    {
+        $transport = $this->createMock(TransportInterface::class);
+
+        $transport->expects($shouldCapture ? $this->once() : $this->never())
+            ->method('send')
+            ->with($this->callback(function (Event $event) use ($thrown): bool {
+                $this->assertCount(1, $event->getExceptions());
```

Comment:
We're not really interested in how many exceptions there are but just that there is at least one, so we can use `assertNotEmpty`